docs/examples/spatial_transformation/SpatialTransformations.jl

```diff
@@ -0,0 +1,94 @@
+# ---
+# title: Cropping, Resizing and Rescaling
+# cover: assets/lighthouse.png
+# author: Ashwani Rathee
+# date: 2020-11-24
+# ---
+
+# This demonstration shows how to use cropping,resizing and rescaling operations on an 
+# image in Julia using ImageTransformations.jl
+
+using ImageCore, ImageShow, ImageTransformations, TestImages, OffsetArrays, ImageShow
```

Comment:
The duplicated `ImageShow` here.

Might be more concise to just import `Images` if we have a lot of packages to import

```suggestion
using Images, TestImages, OffsetArrays
```